precompile/allowlist/config.go

```diff
@@ -10,15 +10,15 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-// Config specifies the initial set of addresses with Admin or Enabled roles.
-type Config struct {
+// AllowList specifies the initial set of addresses with Admin or Enabled roles.
+type AllowList struct {
```

Comment:
we could also leave this as Config and instead of embedding it in other configs name it AllowList there.

Reply:
hm not sure how this would work with embedded jsons. Is there a way to embed jsons without embedding structs? (and definetly not with a custom marshaler again :D)

Reply:
yeah, I went with your above suggestion instead.